File: python/step2/macro.py
import traceback
import json
import re
import os

def handler(event, context):
    macro_response = {
        "requestId": event["requestId"],
        "status": "success"
    }
    # Globals, we only need the fragment in this case but you can get access to all others
    fragment = event['fragment']
    # The extra result copy allows separation of iteration over the fragment and modifying it
    result = fragment

    # This is where you will need to update the code

    # The macro response is what we'll return.
    # This includes the updated fragment piece.
    macro_response['fragment'] = result
    return macro_response

File: node/step2/macro.js
"use strict";

exports.handler = function(event, context, callback) {
    var macro_response = {
        "requestId": event["requestId"],
        "status": "success",
        "fragment": event["fragment"]
    };
    var fragment = event['fragment'];
    var result = fragment;

    macro_response['fragment'] = result;
    callback(null, macro_response);
};